### DISCUSSION TITLE
Run pyzo through source code, how to also display pyzo icon in taskbar？

### DISCUSSION THREAD

**Discussion by wilikeit** [pyzo/pyzo#1081] [Q&A]
D:\r\python\ide\pyzo_venv\Scripts>.\pip install pyside6 pyzo -i https://mirrors.aliyun.com/pypi/simple/  
Scripts>.\pyzo.exe

But after running, the following icon is displayed
![image](https://github.com/user-attachments/assets/883a14cd-e2e5-4425-8e3a-e7576183cf13)

**Comment by bdieterm** [comment]
I am not sure what you are doing here.
So, you are installing PySide6 and Pyzo via pip.
But then you run ".\pyzo.exe" in a directory called "Scripts".
It looks like you are running Pyzo from source. This is why you get the icon of "python.exe" in the taskbar.

If you want the normal Pyzo icon, then run Pyzo as an executable, for example from the following archive:
https://github.com/pyzo/pyzo/releases/download/v4.16.0/pyzo-4.16.0-win64.zip
Or you can build Pyzo.exe yourself using PyInstaller.

**Reply by wilikeit** [reply]
Okay, thank you.I try to using PyInstaller.

**Reply by bdieterm** [reply]
I just updated the building instructions. This might be useful for you:
https://github.com/pyzo/pyzo#building-your-own-binary-executable-of-pyzo

**Reply by wilikeit** [reply]
Okay, that's very helpful.
